Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -1482,28 +1497,25 @@ def _attach_algorithm_func_as_method(method_name, algorithm_wrapper, algm_object
 # but we need to do this earlier
 setattr(mantid, MODULE_NAME, sys.modules['mantid.{}'.format(MODULE_NAME)])
 try:
-    _plugins_key = 'python.plugins.directories'
-    _user_key = 'user.%s' % _plugins_key
```

Comment:
We need to keep the old search behaviour for the key `user.python.plugins.directories` as this is how users can write extensions that are found on startup. If the key is not empty can we do the search after we loaded the "shipped" ones from the manifest? Note that the `user.python.plugins.directories` key can contain multiple `;`-separated values to be treated as a list.